README.md

```diff
@@ -11,20 +11,20 @@ A library for Simple & Efficient data access in Scala and Scala.js
 
 Add the following dependency to your project's build file.
 
+For Scala 2.11.x:
+
 ```scala
-"com.fortysevendeg" %% "fetch" %% "0.2.0"
+"com.fortysevendeg" %% "fetch" % "0.2.0"
```

Comment:
I'm guessing that these versions will change once a new build is made from this PR.